package.json

```diff
@@ -12,36 +12,40 @@
         "ionic:serve": "ionic-app-scripts serve"
     },
     "dependencies": {
-        "@angular/common": "^4.1.3",
-        "@angular/compiler": "^4.1.3",
-        "@angular/compiler-cli": "^4.1.3",
-        "@angular/core": "^4.1.3",
-        "@angular/forms": "^4.1.3",
-        "@angular/http": "^4.1.3",
-        "@angular/platform-browser": "^4.1.3",
-        "@angular/platform-browser-dynamic": "^4.1.3",
-        "@ionic-native/core": "^3.12.1",
-        "@ionic-native/geolocation": "^4.2.1",
-        "@ionic-native/google-maps": "^4.2.1",
+        "@angular/common": "^4.4.4",
+        "@angular/compiler": "^4.4.4",
+        "@angular/compiler-cli": "^4.4.4",
+        "@angular/core": "^4.4.4",
+        "@angular/forms": "^4.4.4",
+        "@angular/http": "^4.4.4",
+        "@angular/platform-browser": "^4.4.4",
+        "@angular/platform-browser-dynamic": "^4.4.4",
+        "@ionic-native/core": "^4.3.0",
+        "@ionic-native/deeplinks": "^4.3.0",
+        "@ionic-native/geolocation": "^4.3.0",
```

Comment:
I wonder if this (upgrading geolocation) is the reason why geolocation doesn't work on android anymore?

Reply:
Good point. I'll try it out.

Reply:
Downgrading to 4.1.0 did not resolve the issue.